Makefile

```diff
@@ -61,9 +61,10 @@ install: venv-create
 	. $(VENV_ACTIVATE_FILE); pip3 install tox
 	# install coverage
 	. $(VENV_ACTIVATE_FILE); pip3 install coverage
-	# also install development dependencies
+	# also install development/release dependencies
 	# workaround for https://github.com/elastic/rally/issues/439
-	. $(VENV_ACTIVATE_FILE); pip3 install -q sphinx sphinx_rtd_theme
+	. $(VENV_ACTIVATE_FILE); pip3 install -q sphinx sphinx_rtd_theme twin wheel
```

Comment:
It's called `twine`

Reply:
Ugh thanks for catching. Fixed in ecdf3c74d3a8d49f1febfdebd496c38fb0b301b9